doc/source/bindings.rst

```diff
@@ -291,6 +291,23 @@ Controls that must be used with LOCK
 Time
 ----
 
+MISSIONTIME, or MET
```

Comment:
This would suggest that `MET` is a valid alternative to `MISSIONTIME`, but looking at the code and in-game `MET` does not exist.